tokenizer/README.md

```diff
@@ -123,3 +123,8 @@ evaluator.evaluateStanfordTokenizer(EvaluationCriteria.ON_SAMPLE_AGAINST_GOLD_ST
 ##Citation
 
 Thank you for citing us if you use us in your work!
+
+```
+@inproceedings{
```

Comment:
If you use this code in your research, please provide the URL for this 
github repository in the relevant publications.